2022-03-03 15:46:57,894 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 15:47:11,313 INFO server.py Установлено соедение с ПК ('127.0.0.1', 54944)
2022-03-03 15:47:11,313 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646311631.3136873, 'user': {'account_name': 'Vasya'}}
2022-03-03 15:47:12,028 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646311631.5245974, 'account_name': 'Vasya'}
2022-03-03 15:47:12,538 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646311632.1680717, 'user': 'Vasya'}
2022-03-03 15:47:20,187 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646311639.8101575, 'account_name': 'Vasya'}
2022-03-03 15:47:20,265 INFO server.py Клиент Vasya корректно отключился от сервера.
2022-03-03 15:50:03,981 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 15:50:15,611 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 15:50:34,155 INFO server.py Установлено соедение с ПК ('127.0.0.1', 54957)
2022-03-03 15:50:34,155 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646311834.155057, 'user': {'account_name': 'Vasya123'}}
2022-03-03 15:50:34,895 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646311834.3852253, 'account_name': 'Vasya123'}
2022-03-03 15:50:35,405 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646311835.0369442, 'user': 'Vasya123'}
2022-03-03 15:50:41,541 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646311841.39164, 'account_name': 'Vasya123'}
2022-03-03 15:50:41,643 INFO server.py Клиент Vasya123 корректно отключился от сервера.
2022-03-03 15:55:58,819 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-03 15:55:59,012 DEBUG descrptrs.py 
2022-03-03 15:55:59,014 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 15:56:24,172 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-03 15:56:24,300 DEBUG descrptrs.py 
2022-03-03 15:56:24,302 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 15:56:43,548 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-03 15:56:43,683 DEBUG descrptrs.py 
2022-03-03 15:56:43,684 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 15:57:07,071 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-03 15:57:07,191 DEBUG descrptrs.py 
2022-03-03 15:57:07,192 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 15:57:07,571 INFO server.py Установлено соедение с ПК ('127.0.0.1', 54996)
2022-03-03 15:57:07,617 INFO server.py Установлено соедение с ПК ('127.0.0.1', 54997)
2022-03-03 15:57:07,617 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646312227.571246, 'user': {'account_name': 'Test0'}}
2022-03-03 15:57:08,313 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646312227.8091612, 'account_name': 'Test0'}
2022-03-03 15:57:08,318 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646312227.634056, 'user': {'account_name': 'Test1'}}
2022-03-03 15:57:39,333 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-03 15:57:39,415 DEBUG descrptrs.py 
2022-03-03 15:57:39,417 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 15:57:39,734 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55001)
2022-03-03 15:57:39,920 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55000)
2022-03-03 15:57:39,921 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646312259.7497034, 'user': {'account_name': 'test2'}}
2022-03-03 15:57:40,479 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646312259.9749906, 'account_name': 'test2'}
2022-03-03 15:57:40,485 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646312259.9368937, 'user': {'account_name': 'test1'}}
2022-03-03 15:57:41,148 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646312260.6711948, 'user': 'test2'}
2022-03-03 15:57:41,151 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646312260.6360085, 'account_name': 'test1'}
2022-03-03 15:57:41,664 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646312261.3264072, 'user': 'test1'}
2022-03-03 15:58:01,599 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1646312281.1830833, 'user': 'test1', 'account_name': 'test2'}
2022-03-03 15:58:14,976 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1646312294.4119885, 'user': 'test2', 'account_name': 'test1'}
2022-03-03 15:58:26,819 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1646312306.4455392, 'mess_text': 'privet'}
2022-03-03 16:01:55,294 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-03 16:01:55,419 DEBUG descrptrs.py 
2022-03-03 16:01:55,421 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 16:01:55,836 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55019)
2022-03-03 16:01:55,883 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55020)
2022-03-03 16:01:55,883 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646312515.852046, 'user': {'account_name': 'test2'}}
2022-03-03 16:01:56,476 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646312515.9761362, 'account_name': 'test2'}
2022-03-03 16:01:56,482 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646312515.898277, 'user': {'account_name': 'test1'}}
2022-03-03 16:01:57,144 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646312516.6727502, 'user': 'test2'}
2022-03-03 16:01:57,147 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646312516.637703, 'account_name': 'test1'}
2022-03-03 16:01:57,660 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646312517.3033156, 'user': 'test1'}
2022-03-03 16:02:04,306 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1646312524.0465157, 'mess_text': 'privet'}
2022-03-03 16:05:43,568 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-03 16:05:43,686 DEBUG descrptrs.py 
2022-03-03 16:05:43,688 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 16:05:44,073 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55040)
2022-03-03 16:05:44,135 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55041)
2022-03-03 16:05:44,136 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646312744.088144, 'user': {'account_name': 'test2'}}
2022-03-03 16:05:44,725 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646312744.2211008, 'account_name': 'test2'}
2022-03-03 16:05:44,730 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646312744.151481, 'user': {'account_name': 'test1'}}
2022-03-03 16:05:45,377 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646312744.908434, 'user': 'test2'}
2022-03-03 16:05:45,380 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646312744.873329, 'account_name': 'test1'}
2022-03-03 16:05:45,894 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646312745.5980444, 'user': 'test1'}
2022-03-03 16:05:54,059 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1646312754.0456557, 'mess_text': 'privet'}
2022-03-03 16:06:36,704 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-03 16:06:36,809 DEBUG descrptrs.py 
2022-03-03 16:06:36,811 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 16:06:37,214 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55043)
2022-03-03 16:06:37,228 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55042)
2022-03-03 16:06:37,743 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646312797.2287111, 'user': {'account_name': 'test1'}}
2022-03-03 16:06:37,851 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646312797.2287111, 'user': {'account_name': 'test2'}}
2022-03-03 16:06:38,449 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646312797.8533533, 'account_name': 'test1'}
2022-03-03 16:06:38,451 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646312797.9439666, 'account_name': 'test2'}
2022-03-03 16:06:38,961 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646312798.607014, 'user': 'test1'}
2022-03-03 16:06:38,964 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646312798.607014, 'user': 'test2'}
2022-03-03 16:06:42,529 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1646312802.0448244, 'mess_text': 'fdsfsd'}
2022-03-03 16:13:53,964 DEBUG server.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2022-03-03 16:13:54,064 DEBUG descrptrs.py 
2022-03-03 16:13:54,065 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 16:13:54,445 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55084)
2022-03-03 16:13:54,491 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55085)
2022-03-03 16:13:54,492 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646313234.4451008, 'user': {'account_name': 'test1'}}
2022-03-03 16:13:55,109 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646313234.6008048, 'account_name': 'test1'}
2022-03-03 16:13:55,116 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646313234.507815, 'user': {'account_name': 'test2'}}
2022-03-03 16:13:55,753 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646313235.2781115, 'user': 'test1'}
2022-03-03 16:13:55,756 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646313235.2519462, 'account_name': 'test2'}
2022-03-03 16:13:56,267 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646313235.9175653, 'user': 'test2'}
2022-03-03 16:14:00,859 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1646313240.239478, 'mess_text': 'fdsfds'}
2022-03-03 16:31:42,803 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 16:31:42,911 DEBUG descrptrs.py 
2022-03-03 16:31:42,912 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 16:31:43,237 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55508)
2022-03-03 16:31:43,252 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55509)
2022-03-03 16:31:43,253 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646314303.2373037, 'user': {'account_name': 'test1'}}
2022-03-03 16:31:43,858 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646314303.3454063, 'account_name': 'test1'}
2022-03-03 16:31:43,865 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646314303.2681208, 'user': {'account_name': 'test2'}}
2022-03-03 16:31:44,527 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646314304.0404499, 'user': 'test1'}
2022-03-03 16:31:44,531 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646314304.0153522, 'account_name': 'test2'}
2022-03-03 16:31:45,041 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646314304.7175522, 'user': 'test2'}
2022-03-03 16:31:49,625 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1646314308.792517, 'mess_text': 'fdfd'}
2022-03-03 16:35:01,553 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 16:35:01,667 DEBUG descrptrs.py 
2022-03-03 16:35:01,669 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 16:35:01,996 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55526)
2022-03-03 16:35:02,043 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55527)
2022-03-03 16:35:02,044 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646314501.9964967, 'user': {'account_name': 'test2'}}
2022-03-03 16:35:02,666 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646314502.159357, 'account_name': 'test2'}
2022-03-03 16:35:02,672 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646314502.0434916, 'user': {'account_name': 'test1'}}
2022-03-03 16:35:03,352 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646314502.8766122, 'user': 'test2'}
2022-03-03 16:35:03,355 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646314502.8423722, 'account_name': 'test1'}
2022-03-03 16:35:03,867 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646314503.5682518, 'user': 'test1'}
2022-03-03 16:35:16,106 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1646314515.9936612, 'mess_text': 'fdsfsd'}
2022-03-03 16:36:10,397 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 16:36:10,496 DEBUG descrptrs.py 
2022-03-03 16:36:10,497 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 16:36:10,852 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55540)
2022-03-03 16:36:10,913 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55541)
2022-03-03 16:36:10,914 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646314570.868235, 'user': {'account_name': 'test2'}}
2022-03-03 16:36:11,642 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646314571.1319773, 'account_name': 'test2'}
2022-03-03 16:36:11,649 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646314570.9285262, 'user': {'account_name': 'test1'}}
2022-03-03 16:36:17,529 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 16:36:17,679 DEBUG descrptrs.py 
2022-03-03 16:36:17,681 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 16:36:17,980 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55543)
2022-03-03 16:36:18,025 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55544)
2022-03-03 16:36:18,026 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646314577.9956765, 'user': {'account_name': 'test1'}}
2022-03-03 16:36:18,745 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646314578.2405548, 'account_name': 'test1'}
2022-03-03 16:36:18,750 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646314578.0259953, 'user': {'account_name': 'test2'}}
2022-03-03 16:36:19,430 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646314578.9664466, 'user': 'test1'}
2022-03-03 16:36:19,433 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646314578.924126, 'account_name': 'test2'}
2022-03-03 16:36:19,948 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646314579.5934603, 'user': 'test2'}
2022-03-03 16:36:25,571 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1646314585.0044062, 'user': 'test1', 'account_name': 'test2'}
2022-03-03 16:36:30,287 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1646314589.5392795, 'mess_text': 'gfdgfd'}
2022-03-03 16:36:30,414 INFO server.py Отправлено сообщение пользователю test2 от пользователя test1.
2022-03-03 16:36:35,543 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1646314595.302075, 'user': 'test2', 'account_name': 'test1'}
2022-03-03 16:36:39,240 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1646314598.8190866, 'mess_text': 'fdsfdsf'}
2022-03-03 16:36:39,364 INFO server.py Отправлено сообщение пользователю test1 от пользователя test2.
2022-03-03 16:36:58,224 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646314617.8581781, 'account_name': 'test2'}
2022-03-03 16:36:58,336 INFO server.py Клиент test2 корректно отключился от сервера.
2022-03-03 16:36:58,850 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646314618.671596, 'account_name': 'test1'}
2022-03-03 16:36:58,936 INFO server.py Клиент test1 корректно отключился от сервера.
2022-03-03 16:38:03,449 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 16:38:03,576 DEBUG descrptrs.py 
2022-03-03 16:38:03,577 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 16:38:03,947 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55561)
2022-03-03 16:38:03,978 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55562)
2022-03-03 16:38:03,979 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646314683.9471517, 'user': {'account_name': 'test1'}}
2022-03-03 16:38:04,604 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646314684.0960035, 'account_name': 'test1'}
2022-03-03 16:38:04,610 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646314683.9782348, 'user': {'account_name': 'test2'}}
2022-03-03 16:38:05,270 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646314684.8127685, 'user': 'test1'}
2022-03-03 16:38:05,273 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646314684.7704706, 'account_name': 'test2'}
2022-03-03 16:38:05,785 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646314685.4383073, 'user': 'test2'}
2022-03-03 16:44:02,768 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 16:44:02,858 DEBUG descrptrs.py 
2022-03-03 16:44:02,860 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 16:44:03,198 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55696)
2022-03-03 16:44:03,230 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55697)
2022-03-03 16:44:03,230 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646315043.214254, 'user': {'account_name': 'test1'}}
2022-03-03 16:44:03,853 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646315043.3448522, 'account_name': 'test1'}
2022-03-03 16:44:03,859 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646315043.245467, 'user': {'account_name': 'test2'}}
2022-03-03 16:44:04,520 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646315044.0383666, 'user': 'test1'}
2022-03-03 16:44:04,524 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646315044.0122623, 'account_name': 'test2'}
2022-03-03 16:44:05,032 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646315044.69501, 'user': 'test2'}
2022-03-03 16:44:13,212 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1646315052.7502463, 'mess_text': 'fdsfdsfs'}
2022-03-03 16:44:13,317 INFO server.py Отправлено сообщение пользователю test2 от пользователя test1.
2022-03-03 16:44:24,018 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646315063.563888, 'account_name': 'test2'}
2022-03-03 16:44:24,066 INFO server.py Клиент test2 корректно отключился от сервера.
2022-03-03 16:44:25,596 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646315065.1253133, 'account_name': 'test1'}
2022-03-03 16:44:25,682 INFO server.py Клиент test1 корректно отключился от сервера.
2022-03-03 16:45:46,158 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 16:45:46,272 DEBUG descrptrs.py 
2022-03-03 16:45:46,274 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 16:45:46,558 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55701)
2022-03-03 16:45:46,605 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55702)
2022-03-03 16:45:46,606 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646315146.5741687, 'user': {'account_name': 'test2'}}
2022-03-03 16:45:47,228 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646315146.7256794, 'account_name': 'test2'}
2022-03-03 16:45:47,234 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646315146.6052039, 'user': {'account_name': 'test1'}}
2022-03-03 16:45:47,893 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646315147.41747, 'user': 'test2'}
2022-03-03 16:45:47,896 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646315147.3838754, 'account_name': 'test1'}
2022-03-03 16:45:48,410 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646315148.0592768, 'user': 'test1'}
2022-03-03 16:45:59,140 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646315158.3950748, 'account_name': 'test2'}
2022-03-03 16:45:59,255 INFO server.py Клиент test2 корректно отключился от сервера.
2022-03-03 16:46:24,311 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 16:46:24,413 DEBUG descrptrs.py 
2022-03-03 16:46:24,414 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 16:46:24,852 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55705)
2022-03-03 16:46:24,853 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646315184.8523786, 'user': {'account_name': 'test1'}}
2022-03-03 16:46:24,973 INFO server.py Установлено соедение с ПК ('127.0.0.1', 55706)
2022-03-03 16:46:24,974 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646315184.8837693, 'user': {'account_name': 'test2'}}
2022-03-03 16:46:25,600 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646315184.9744356, 'account_name': 'test1'}
2022-03-03 16:46:25,605 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646315185.1013408, 'account_name': 'test2'}
2022-03-03 16:46:26,110 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646315185.7660563, 'user': 'test1'}
2022-03-03 16:46:26,114 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646315185.7660563, 'user': 'test2'}
2022-03-03 16:46:37,364 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646315197.20389, 'account_name': 'test2'}
2022-03-03 16:46:37,471 INFO server.py Клиент test2 корректно отключился от сервера.
2022-03-03 16:46:37,986 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646315197.759547, 'account_name': 'test1'}
2022-03-03 16:46:38,104 INFO server.py Клиент test1 корректно отключился от сервера.
2022-03-03 16:50:42,668 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 16:50:42,792 DEBUG descrptrs.py 
2022-03-03 16:50:42,793 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 16:50:43,103 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56102)
2022-03-03 16:50:43,134 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56103)
2022-03-03 16:50:43,134 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646315443.118576, 'user': {'account_name': 'test2'}}
2022-03-03 16:50:43,766 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646315443.2598112, 'account_name': 'test2'}
2022-03-03 16:50:43,771 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646315443.1341364, 'user': {'account_name': 'test1'}}
2022-03-03 16:50:44,427 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646315443.9593575, 'user': 'test2'}
2022-03-03 16:50:44,431 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646315443.925942, 'account_name': 'test1'}
2022-03-03 16:50:44,936 INFO server.py Клиент ('127.0.0.1', 56102) отключился от сервера.
2022-03-03 16:50:44,981 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646315444.5937557, 'user': 'test1'}
2022-03-03 16:50:45,494 INFO server.py Клиент ('127.0.0.1', 56103) отключился от сервера.
2022-03-03 16:50:58,367 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 16:50:58,472 DEBUG descrptrs.py 
2022-03-03 16:50:58,474 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 16:50:58,873 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56123)
2022-03-03 16:50:58,874 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56124)
2022-03-03 16:50:58,875 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646315458.873712, 'user': {'account_name': 'test1'}}
2022-03-03 16:50:59,000 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646315458.873712, 'user': {'account_name': 'test2'}}
2022-03-03 16:50:59,616 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646315459.001105, 'account_name': 'test1'}
2022-03-03 16:50:59,621 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646315459.1089485, 'account_name': 'test2'}
2022-03-03 16:51:00,126 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646315459.8010814, 'user': 'test1'}
2022-03-03 16:51:00,129 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646315459.8010814, 'user': 'test2'}
2022-03-03 16:56:15,501 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 16:56:15,605 DEBUG descrptrs.py 
2022-03-03 16:56:15,607 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 16:56:15,978 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56571)
2022-03-03 16:56:15,979 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56572)
2022-03-03 16:56:15,980 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646315775.978763, 'user': {'account_name': 'test1'}}
2022-03-03 16:56:16,615 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646315775.9947124, 'user': {'account_name': 'test2'}}
2022-03-03 16:56:16,742 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646315776.1090553, 'account_name': 'test1'}
2022-03-03 16:56:17,252 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646315776.7440805, 'account_name': 'test2'}
2022-03-03 16:56:17,253 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646315776.8517323, 'user': 'test1'}
2022-03-03 16:56:17,768 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646315777.4089658, 'user': 'test2'}
2022-03-03 16:56:24,898 INFO server.py Клиент ('127.0.0.1', 56572) отключился от сервера.
2022-03-03 16:56:26,034 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646315785.7517502, 'account_name': 'test2'}
2022-03-03 16:56:26,122 INFO server.py Клиент test2 корректно отключился от сервера.
2022-03-03 17:01:28,869 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 17:01:28,989 DEBUG descrptrs.py 
2022-03-03 17:01:28,991 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 17:01:29,291 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56718)
2022-03-03 17:01:29,354 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56719)
2022-03-03 17:01:29,355 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646316089.3067796, 'user': {'account_name': 'test2'}}
2022-03-03 17:01:29,975 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646316089.4759085, 'account_name': 'test2'}
2022-03-03 17:01:29,981 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646316089.3703687, 'user': {'account_name': 'test1'}}
2022-03-03 17:01:30,643 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646316090.1704862, 'user': 'test2'}
2022-03-03 17:01:30,646 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646316090.1347985, 'account_name': 'test1'}
2022-03-03 17:01:31,158 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646316090.8340876, 'user': 'test1'}
2022-03-03 17:01:39,821 INFO server.py Клиент ('127.0.0.1', 56718) отключился от сервера.
2022-03-03 17:01:47,597 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646316107.1042101, 'account_name': 'test1'}
2022-03-03 17:01:47,705 INFO server.py Клиент test1 корректно отключился от сервера.
2022-03-03 17:09:54,230 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 17:09:54,350 DEBUG descrptrs.py 
2022-03-03 17:09:54,352 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 17:09:54,666 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56881)
2022-03-03 17:09:54,667 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646316594.6663146, 'user': {'account_name': 'test1'}}
2022-03-03 17:09:54,794 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56882)
2022-03-03 17:09:54,795 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646316594.727153, 'user': {'account_name': 'test2'}}
2022-03-03 17:09:55,428 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646316594.7964141, 'account_name': 'test1'}
2022-03-03 17:09:55,434 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646316594.9210715, 'account_name': 'test2'}
2022-03-03 17:09:55,943 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646316595.6142063, 'user': 'test1'}
2022-03-03 17:09:55,946 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646316595.6142063, 'user': 'test2'}
2022-03-03 17:10:03,099 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1646316602.9058886, 'mess_text': 'fdsfds'}
2022-03-03 17:10:03,220 INFO server.py Отправлено сообщение пользователю test1 от пользователя test2.
2022-03-03 17:10:17,487 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646316616.9100301, 'account_name': 'test1'}
2022-03-03 17:10:17,600 INFO server.py Клиент test1 корректно отключился от сервера.
2022-03-03 17:10:18,113 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646316617.6417723, 'account_name': 'test2'}
2022-03-03 17:10:18,208 INFO server.py Клиент test2 корректно отключился от сервера.
2022-03-03 17:13:58,264 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 17:13:58,430 DEBUG descrptrs.py 
2022-03-03 17:13:58,432 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 17:13:58,729 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56927)
2022-03-03 17:13:58,729 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56926)
2022-03-03 17:13:58,733 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646316838.7298126, 'user': {'account_name': 'test2'}}
2022-03-03 17:13:59,349 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646316838.7451668, 'user': {'account_name': 'test1'}}
2022-03-03 17:13:59,472 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646316838.8475041, 'account_name': 'test2'}
2022-03-03 17:13:59,986 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646316839.4733908, 'account_name': 'test1'}
2022-03-03 17:13:59,987 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646316839.5921612, 'user': 'test2'}
2022-03-03 17:14:00,502 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646316840.1385965, 'user': 'test1'}
2022-03-03 17:14:08,691 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646316848.2045698, 'account_name': 'test1'}
2022-03-03 17:14:08,813 INFO server.py Клиент test1 корректно отключился от сервера.
2022-03-03 17:16:18,769 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 17:16:18,865 DEBUG descrptrs.py 
2022-03-03 17:16:18,867 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 17:16:19,258 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56960)
2022-03-03 17:16:19,259 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56959)
2022-03-03 17:16:19,260 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646316979.2586665, 'user': {'account_name': 'test2'}}
2022-03-03 17:16:19,401 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646316979.2586665, 'user': {'account_name': 'test1'}}
2022-03-03 17:16:20,023 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646316979.4020073, 'account_name': 'test2'}
2022-03-03 17:16:20,029 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646316979.5198581, 'account_name': 'test1'}
2022-03-03 17:16:20,537 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646316980.2032816, 'user': 'test2'}
2022-03-03 17:16:20,539 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646316980.2042856, 'user': 'test1'}
2022-03-03 17:16:21,050 INFO server.py Клиент ('127.0.0.1', 56960) отключился от сервера.
2022-03-03 17:16:21,132 INFO server.py Клиент ('127.0.0.1', 56959) отключился от сервера.
2022-03-03 17:17:36,519 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 17:17:36,666 DEBUG descrptrs.py 
2022-03-03 17:17:36,667 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 17:17:37,086 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56969)
2022-03-03 17:17:37,087 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56970)
2022-03-03 17:17:37,089 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646317057.0865104, 'user': {'account_name': 'test2'}}
2022-03-03 17:17:37,710 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646317057.2085097, 'account_name': 'test2'}
2022-03-03 17:17:37,717 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646317057.1025755, 'user': {'account_name': 'test1'}}
2022-03-03 17:17:38,378 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646317057.9082937, 'user': 'test2'}
2022-03-03 17:17:38,381 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646317057.8674173, 'account_name': 'test1'}
2022-03-03 17:17:38,893 INFO server.py Клиент ('127.0.0.1', 56969) отключился от сервера.
2022-03-03 17:17:38,972 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646317058.5472157, 'user': 'test1'}
2022-03-03 17:17:39,489 INFO server.py Клиент ('127.0.0.1', 56970) отключился от сервера.
2022-03-03 17:18:02,604 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 17:18:02,721 DEBUG descrptrs.py 
2022-03-03 17:18:02,722 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 17:18:03,010 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56977)
2022-03-03 17:18:03,011 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56978)
2022-03-03 17:18:03,012 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646317083.0119417, 'user': {'account_name': 'test1'}}
2022-03-03 17:18:03,648 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646317083.0266244, 'user': {'account_name': 'test2'}}
2022-03-03 17:18:03,774 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646317083.1409712, 'account_name': 'test1'}
2022-03-03 17:18:04,288 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646317083.7757013, 'account_name': 'test2'}
2022-03-03 17:18:04,289 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646317083.8842971, 'user': 'test1'}
2022-03-03 17:18:04,802 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646317084.4244184, 'user': 'test2'}
2022-03-03 17:18:04,803 INFO server.py Клиент ('127.0.0.1', 56978) отключился от сервера.
2022-03-03 17:18:05,454 INFO server.py Клиент ('127.0.0.1', 56977) отключился от сервера.
2022-03-03 17:18:14,527 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 17:18:14,665 DEBUG descrptrs.py 
2022-03-03 17:18:14,667 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 17:18:14,896 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56979)
2022-03-03 17:18:14,897 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646317094.8946798, 'user': {'account_name': 'test1'}}
2022-03-03 17:18:15,014 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56980)
2022-03-03 17:18:15,015 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646317094.9260619, 'user': {'account_name': 'test2'}}
2022-03-03 17:18:15,640 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646317095.0159302, 'account_name': 'test1'}
2022-03-03 17:18:15,646 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646317095.1330917, 'account_name': 'test2'}
2022-03-03 17:18:16,150 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646317095.8272512, 'user': 'test1'}
2022-03-03 17:18:16,153 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646317095.8272512, 'user': 'test2'}
2022-03-03 17:18:16,663 INFO server.py Клиент ('127.0.0.1', 56979) отключился от сервера.
2022-03-03 17:18:16,712 INFO server.py Клиент ('127.0.0.1', 56980) отключился от сервера.
2022-03-03 17:18:26,284 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 17:18:26,395 DEBUG descrptrs.py 
2022-03-03 17:18:26,397 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 17:18:26,736 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56983)
2022-03-03 17:18:26,737 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56984)
2022-03-03 17:18:26,739 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646317106.737068, 'user': {'account_name': 'test1'}}
2022-03-03 17:18:26,864 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646317106.737068, 'user': {'account_name': 'test2'}}
2022-03-03 17:18:27,496 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646317106.8658705, 'account_name': 'test1'}
2022-03-03 17:18:27,503 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646317106.98325, 'account_name': 'test2'}
2022-03-03 17:18:28,008 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646317107.6584044, 'user': 'test1'}
2022-03-03 17:18:28,011 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646317107.6661706, 'user': 'test2'}
2022-03-03 17:18:34,639 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1646317113.8853812, 'mess_text': 'fdsfsd'}
2022-03-03 17:18:34,754 INFO server.py Отправлено сообщение пользователю test2 от пользователя test1.
2022-03-03 17:18:48,477 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646317128.1700044, 'account_name': 'test2'}
2022-03-03 17:18:48,606 INFO server.py Клиент test2 корректно отключился от сервера.
2022-03-03 17:18:49,625 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646317128.9992862, 'account_name': 'test1'}
2022-03-03 17:18:49,719 INFO server.py Клиент test1 корректно отключился от сервера.
2022-03-03 17:19:52,591 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 17:19:52,710 DEBUG descrptrs.py 
2022-03-03 17:19:52,712 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 17:19:53,032 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56989)
2022-03-03 17:19:53,077 INFO server.py Установлено соедение с ПК ('127.0.0.1', 56990)
2022-03-03 17:19:53,077 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646317193.0473793, 'user': {'account_name': 'test1'}}
2022-03-03 17:19:53,709 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646317193.2046185, 'account_name': 'test1'}
2022-03-03 17:19:53,714 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646317193.0928497, 'user': {'account_name': 'test2'}}
2022-03-03 17:19:54,377 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646317193.898106, 'user': 'test1'}
2022-03-03 17:19:54,381 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646317193.8720725, 'account_name': 'test2'}
2022-03-03 17:19:54,891 INFO server.py Клиент ('127.0.0.1', 56989) отключился от сервера.
2022-03-03 17:19:54,976 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646317194.5361118, 'user': 'test2'}
2022-03-03 17:19:55,484 INFO server.py Клиент ('127.0.0.1', 56990) отключился от сервера.
2022-03-03 17:22:26,061 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 17:22:26,197 DEBUG descrptrs.py 
2022-03-03 17:22:26,200 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 17:22:26,603 INFO server.py Установлено соедение с ПК ('127.0.0.1', 57006)
2022-03-03 17:22:26,665 INFO server.py Установлено соедение с ПК ('127.0.0.1', 57007)
2022-03-03 17:22:26,666 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646317346.618638, 'user': {'account_name': 'test1'}}
2022-03-03 17:22:27,303 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646317346.7926488, 'account_name': 'test1'}
2022-03-03 17:22:27,309 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646317346.6653392, 'user': {'account_name': 'test2'}}
2022-03-03 17:23:10,499 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 17:23:10,613 DEBUG descrptrs.py 
2022-03-03 17:23:10,615 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 17:23:11,025 INFO server.py Установлено соедение с ПК ('127.0.0.1', 57011)
2022-03-03 17:23:11,071 INFO server.py Установлено соедение с ПК ('127.0.0.1', 57012)
2022-03-03 17:23:11,071 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646317391.0256507, 'user': {'account_name': 'test1'}}
2022-03-03 17:23:11,709 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646317391.1995845, 'account_name': 'test1'}
2022-03-03 17:23:11,715 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646317391.0876923, 'user': {'account_name': 'test2'}}
2022-03-03 17:24:07,164 DEBUG decorators.py Была вызвана функция arg_parser c параметрами ('7777', '') , {}. Вызов из модуля __main__
2022-03-03 17:24:07,270 DEBUG descrptrs.py 
2022-03-03 17:24:07,271 INFO server.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: .              Если адрес не указан, принимаются соединения с любых адресов.
2022-03-03 17:24:07,728 INFO server.py Установлено соедение с ПК ('127.0.0.1', 57015)
2022-03-03 17:24:07,729 INFO server.py Установлено соедение с ПК ('127.0.0.1', 57014)
2022-03-03 17:24:07,732 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646317447.7287114, 'user': {'account_name': 'test2'}}
2022-03-03 17:24:08,362 DEBUG server.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1646317447.7445943, 'user': {'account_name': 'test1'}}
2022-03-03 17:24:08,472 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646317447.8577304, 'account_name': 'test2'}
2022-03-03 17:24:08,982 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1646317448.4730816, 'account_name': 'test1'}
2022-03-03 17:24:08,982 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646317448.5895152, 'user': 'test2'}
2022-03-03 17:24:09,495 DEBUG server.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1646317449.1318598, 'user': 'test1'}
2022-03-03 17:24:17,155 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1646317457.0764549, 'mess_text': 'fdsfsdf\n'}
2022-03-03 17:24:17,347 INFO server.py Отправлено сообщение пользователю test1 от пользователя test2.
2022-03-03 17:24:21,420 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1646317461.0713427, 'mess_text': 'gfdgdfgdf'}
2022-03-03 17:24:21,547 INFO server.py Отправлено сообщение пользователю test2 от пользователя test1.
2022-03-03 17:24:30,246 DEBUG server.py Разбор сообщения от клиента : {'action': 'remove', 'time': 1646317469.8802373, 'user': 'test1', 'account_name': 'test2'}
2022-03-03 17:24:33,926 DEBUG server.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1646317473.6060855, 'mess_text': 'gfdgd'}
2022-03-03 17:24:34,047 INFO server.py Отправлено сообщение пользователю test1 от пользователя test2.
2022-03-03 17:24:37,114 DEBUG server.py Разбор сообщения от клиента : {'action': 'add', 'time': 1646317476.8625867, 'user': 'test1', 'account_name': 'test2'}
2022-03-03 17:24:40,288 DEBUG server.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1646317480.0487435, 'account_name': 'test2'}
2022-03-03 17:24:40,393 INFO server.py Клиент test2 корректно отключился от сервера.
